### ISSUE TITLE
Set current date/time stamp in the "First Published" field on first publishing

### ISSUE THREAD

**Issue by alansemenov** [enonic/xp#4477]
Depends on #4424 

The value in this field should be set at the moment of first publishing (together with "Publish From") but - unlike "Publish From" - it should never be overwritten.

When content is unpublished, the value in "First Published" field should be cleared only in the case when current date/time is earlier than date/time in "First Published". This is the case when the item was scheduled for publishing but was unpublished before the content went online. In this case content status should still be "New" and not "Offline"

**Comment by GlennRicaud** [comment]
I am taking over this task since vbr is on holidays.
I will pass this back in progress since I see multiple problems here.

Spec: Contrary to what was defined “publish.from" is not overwritten. Is that supposed to be the case now?
Implementation: In the code of set publish, there is no handling of the case where publish.from is set but not publish.first. It will be the case with contents created in 6.9.
Implementation: Unpublished: I do not see any any implementation of your 'When content is unpublished, the value in "First Published" field should be cleared only in the case when current date/time is earlier than date/time in "First Published”'. 
Implementation: Unpublish: Modified time modified even when not necessary

I worked on the publish.from for 6.9 and Runar on 6.8 and I do not think anyone of us has been part of the meeting to plan this feature. 
I do not find the feature listed in "https://github.com/enonic/enonic”, are we not using it anymore?

So to sum up. Lets have a quick meeting, discuss it before we proceed
